### DISCUSSION TITLE
FIX: MUI TextField Label Doesn't Shrink on setValue

### DISCUSSION THREAD

**Discussion by zeshantech** [react-hook-form/react-hook-form#11423] [Q&A] [answered]
### Version Number

@latest

### Codesandbox/Expo snack

https://codesandbox.io/p/devbox/lc6vlr?file=%2Fsrc%2FApp.tsx%3A14%2C20

### Expected behaviour

Expected behaviour is simple it should be shrink when setValue by react-form-hook (setvalue)

### Code of Conduct
- [x] I agree to follow this project's Code of Conduct

**Comment by zeshantech** [comment]
@everyo ??

**Reply by epicstar** [reply]
It's a known issue that either MUI or react-hook-form: https://github.com/react-hook-form/react-hook-form/issues/220

**Reply by epicstar** [reply, marked as answer]
To clarify, we should should be using a controller with the MUI TextField: https://github.com/mui/material-ui/issues/17018#issuecomment-860282039
